docs/standard/serialization/binaryformatter-migration-guide/winforms-applications.md

```diff
@@ -0,0 +1,98 @@
+
+## How BinaryFormatter affects Windows Forms
+
+Prior to .NET 9, Windows Forms used `BinaryFormatter` to serialize and deserialize data for scenarios such clipboard, drag-and-drop, and storing or loading resources at design time. Starting with .NET 9, Windows Forms and WPF use a subset of the `BinaryFormatter` implementation internally for these scenarios. While BinaryFormatter's risks cannot be addressed in general-purpose serialization/deserialization, measures have been taken to mitigate the risks in these very specific use cases with a known set of types. A fall-back to `BinaryFormatter` is still in place for unknown or unsupported types, which will throw exceptions unless migration steps are taken in the application.
+
+Windows Forms and WPF apps both handle the following types, along with arrays and lists of these types. Clipboard, drag-and-drop, and design-time resources will continue to work with these types without any migration steps needed.
+
+- `bool`
+- `byte`
+- `char`
+- `decimal`
+- `double`
+- `int`
+- `sbyte`
+- `float`
+- `TimeSpan`
+- `DateTime`
+- `uint`
+- `string`
+- `nint`
+- `nuint`
+- `long`
+- `ulong`
+- `short`
+- `ushort`
+- `PointF`
+- `RectangleF`
+
+Windows Forms also supports the following additional types:
+
+- `Bitmap`
+- `ImageListStreamer`
+
+### OLE scenarios
+
+For information about the effects BinaryFormatter removal has on OLE scenarios such as clipboard and drag-and-drop as well as migration guidance see [Windows Forms and Windows Presentation Foundation BinaryFormatter OLE guidance](./winforms-wpf-ole-guidance.md).
+
+### Resources (ResX)
+
+#### The Windows Forms Designer
+
+The Windows Forms Out-Of-Process Designer also uses `BinaryFormatter` internally for ResX serialization and deserialization.
+
+Types and properties might participate in serialization without you realizing due to the standard behavior of the Windows Forms Designer. One way that BinaryFormatter is used that you might not be aware of is when a `public` property on a <xref:System.ComponentModel.IComponent> is introduced and that component is populated or that property is edited at design time. It's likely that that data gets serialized into resource files. Consider the following conditions:
+
+- A public property contains data at the time when a `Form` in the Designer is saved.
+- That property is not read-only.
+- That property is not attributed with `DesignerSerializationVisibility(false)`.
+- That property does not have a DefaultValueAttribute.
+- That property does not have a respective `bool ShouldSerialize[PropertyName]` method that returns `false` at the time of the CodeDOM serialization process. (Note: the method can have `private` scope.)
+- That property does not have a [`DesignerSerializer`](/dotnet/api/microsoft.visualstudio.modeling.dsldefinition.designerserializer)
```

Comment:
@Tanya-Solyanik this is where new bullet was added based off of your previous suggestion